kolibri/plugins/learn/assets/src/views/cards/QuizCard/index.vue

```diff
@@ -51,11 +51,13 @@
     },
     data() {
       return {
-        progress: this.quiz ? this.quiz.progress : undefined,
         title: this.quiz ? this.quiz.title : '',
```

Comment:
Maybe for consistency we could do the same for the title here? Seems like using a computed prop for this kind of conditional check is more reliable in general.